src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -169,6 +203,16 @@
     </{{ element|default('div') }}>
 {%- endblock form_row %}
 
+{# Help #}
+
+{% block form_help -%}
+    {% for attrname, attrvalue in attr %}
+        {% if attrname == 'data-help' %}
+            <small class="form-text text-muted">{{ attrvalue|raw }}</small>
+        {% endif %}
+    {% endfor %}
+{%- endblock form_help %}
+
```

Comment:
I would prefer a real `FormType`-"help"-Option instead of this `data-help`-workaround.

Adding that might be out of scope for this PR, though, because it has nothing to do with Bootstrap at all and would have to be backported to the `form_div_layout.html.twig`?